pjp.tm08/src/main/java/com/subho/wipro/pjp/tm08/UsingPreparedStatementAndMetaDataObjects/Assignment1.java

```diff
@@ -0,0 +1,58 @@
+package com.subho.wipro.pjp.tm08.UsingPreparedStatementAndMetaDataObjects;
+
+public class Assignment1 {
+	
+	public static void main(String[] args) {
+		int option = Integer.parseInt(args[0]);
+		DAOClass dao = new DAOClass();
+		
+		switch(option) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Switch statements should have a default label](https://app.codacy.com/manual/ohbus/PJP/pullRequest?prid=5628071)